bpdv3/Assets/Scripts/UI/Main/BeatBar.cs

```diff
@@ -42,6 +48,7 @@ void Start()
         // Update is called once per frame
         void Update()
         {
+            threshHoldValue = thresholdZone;
```

Comment:
oops, i thought i removed this. I need to remove this. I made just so i could see the static variables value in the inspector.